src/main/scala/com/itechart/ny_accidents/service/metric/DistrictMetricService.scala

```diff
@@ -1,11 +1,39 @@
 package com.itechart.ny_accidents.service.metric
 
-import com.itechart.ny_accidents.entity.{District, MergedData, ReportMergedData}
+import com.itechart.ny_accidents.entity.{DetailedDistrictData, District, MergedData}
 import com.itechart.ny_accidents.spark.Spark
 import org.apache.spark.rdd.RDD
 
 object DistrictMetricService extends PercentageMetricService {
 
+  def getDetailedDistrictData(data: RDD[MergedData]):  RDD[DetailedDistrictData] = {
```

Comment:
Good candidate for unit test

Reply:
@sleshJdev 
this method already has a test
https://github.com/itechartbigdatalab/NY-Accidents-Rate/commit/97e82c0af1e43505f53dddb4e7783881a5acb3c7